common/src/java/com/zimbra/common/util/Log.java

```diff
@@ -287,6 +287,14 @@ public void warn(Object o) {
     public void warn(Object o, Throwable t) {
         getLogger().warn(o, t);
     }
+    
+    public void warnQuietly(Object o, Throwable t) {
+        if (isDebugEnabled()) {
+            getLogger().debug(o, t);
```

Comment:
Please change this (back) to warn. (same for other one lower down)
This method is intended to be used for warnings, so the logged result should reflect that, grepping for WARN should find it.  The fact that a more detailed warning is emitted because debug is enabled is irrelevant.

Reply:
Fixed